lib/lita/handlers/bumpbot_handler.rb

```diff
@@ -73,23 +121,27 @@ def self.command_route(command, method_sym, help, max_args = 0)
       def handle_event(title)
         init_event(title)
         yield
+        cleanup
       rescue ErrorAlreadyReported
-        error("Aborting message handler \"#{title}\" due to previously raised error")
+        debug("Sandbox: #{sandbox_directory}")
       rescue
         msg = "Unhandled error while working on \"#{title}\":\n" +
-          "```#{$!}\n#{$!.backtrace.join("\n")}"
+          "```#{$!}\n#{$!.backtrace.join("\n")}."
+        debug("Sandbox: #{sandbox_directory}")
         error(msg)
       end
 
       def run_command(command, timeout: 3600, **options)
         Bundler.with_clean_env do
           options[:timeout] = timeout
 
+          command = Shellwords.join(command) if command.is_a?(Array)
           debug("Running \"#{command}\" with #{options}")
           shellout = Mixlib::ShellOut.new(command, options)
           shellout.run_command
-          debug("STDOUT:#{shellout.stdout}\nSTDERR:#{shellout.stderr}")
           shellout.error!
+          debug("STDOUT:\n```#{shellout.stdout}```\n")
```

Comment:
I wanted at least one line to be printed, actually. I thought it would look like the bot wasn't working if it didn't print anything at all.